platform/generic/manifest.mf

```diff
@@ -0,0 +1 @@
+OpenIDE-Module: generic
```

Comment:
The name is not good.

Reply:
Renamed to `org.netbeans.html.presenters.spi`.